src/script/team/TeamPermission.js

```diff
@@ -17,89 +17,101 @@
  *
  */
 
-window.z = window.z || {};
-window.z.team = z.team || {};
+/**
+ * Enum for different team permissions.
+ * @returns {z.team.TeamPermission.PERMISSION} Enum of team permissions
+ */
+const PERMISSION = {
```

Comment:
Can we sort that by value and not alphabetical, please?

Reply:
I wouldn't say value is a better sorting algo than feature name....
Why do you feel it's better?

Reply:
Maybe it's just my OCD kicking in, but I feel that with bitflags the value is way more important than the name that you chose for it.